pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -416,14 +416,28 @@ func (i *InfraProvider) ExtractHostAddresses(dir string, config *types.InstallCo
 	}
 	logrus.Debugf("bootstrap manifests found: %v", bootstrapFiles)
 
-	if len(bootstrapFiles) != 1 {
-		return fmt.Errorf("wrong number of bootstrap manifests found: %v. Expected exactly one", bootstrapFiles)
+	// XXX: why?
```

Comment:
there's only one bootstrap machine so it would make sense to be only one manifest. could be a mistaken assumption or a bug. i will look into it a bit